app/views/merchants/index.html.erb

```diff
@@ -3,7 +3,11 @@
 <section class = "grid-container">
   <% @merchants.each do |merchant|%>
     <section class = "grid-item">
-      <h2><%=link_to merchant.name, "/merchants/#{merchant.id}"%></h2>
+      <% if current_user && current_user.admin? %>
```

Comment:
on merchants index page admin sees a link to see the merchant dashboard using path "/admin/merchants/merchant_id"